components/runtime/squeak4/Makefile

```diff
@@ -0,0 +1,75 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"). You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
```

Comment:
Please, add author here

Reply:
I suppose, you missed
```
#
# Copyright 2020 David Stes
#
```

Reply:
What exactly do you mean by add author ? The Makefile author ?  I'm not sure I understand what you mean by "you missed".   Also you have quoted a piece of code that states that a full copy of the CDDL should accompany the source.  I can copy CDDL.license into the directory (of the Makefile) but obviously the CDDL is not related to the squeak application.  Please clarify as I don't fully understand the reasoning.